README.md

```diff
@@ -36,6 +36,8 @@ Browse and query your MongoDB databases both locally and in the cloud using [_sc
 - Enter your commands and/or comments, eg: `db.<collectionName>.find()`

```

Comment:
Modify the "Right-click on a database and select "Connect" to instead suggest they use the "Connected to" code lens button?